docs/pages/reference/agent-services/mcp-access.mdx

```diff
@@ -0,0 +1,109 @@
+---
+title: MCP Access Reference
+description: Configuration and CLI reference for Teleport MCP access.
+---
+
+This guide describes interfaces and options for interacting with the Teleport
+Application Service for MCP access, including the static configuration file for
+the `teleport` binary, and `tsh mcp` commands.
+
+## Configuration
+
+(!docs/pages/includes/backup-warning.mdx!)
+
+The following snippet shows the full YAML configuration of an Application Service
+appearing in the `teleport.yaml` configuration file:
+
+```yaml
+app_service:
+  # Enables application proxy service.
+  enabled: true
+  # Enables the builtin Teleport demo MCP server that shows current user and
+  # session information. Wildcard must be allowed for "app_labels" by your role
+  # to access this MCP server.
+  mcp_demo_server: true
+  # This section contains definitions of all applications proxied by this
+  # service. It can contain multiple items.
+  apps:
+  # Name of the application. Used for identification purposes.
+  - name: "mcp-everything"
+    # Free-form application description.
+    description: "Example Everything MCP server"
+    # Static labels to assign to the app. Used in RBAC.
+    labels:
+      env: "prod"
+    # Contains MCP server-related configurations.
+    mcp:
+      # Command to launch stdio-based MCP servers.
+      command: "docker"
+      # Args to execute with the command.
+      args: ["run", "-i", "--rm", "mcp/everything"]
+      # Name of the host user account under which the command will be
+      # executed. Required for stdio-based MCP servers.
+      run_as_host_user: "docker"
```

Comment:
Similar to my question above, can we also include an example of the MCP server yaml resource in the reference you would create with tctl/API.